src/components/TeamPage.tsx

```diff
@@ -36,6 +39,14 @@ export const TeamPage = ({ teamId }: TeamPageProps) => {
                         children.map((child) => <GroupTreeViewNode key={child.id} group={child} visible />),
                     )}
                 </TreeView>
+                <PreviewContent>
```

Comment:
You don't need PreviewContent here, it's not a preview.

Just add local styles if you need gaps between components